corehq/apps/linked_domain/templates/linked_domain/get_started.html

```diff
@@ -2,30 +2,35 @@
 {% load i18n %}
 
 <div id="ko-getting-started" data-bind="with: $root.gettingStartedViewModel">
-  <h3>{% trans "Linked Projects" %}</h3>
+  <h3>{% trans "Linked Project Spaces" %}</h3>
+  <p>{% trans "Create links between upstream and downstream project spaces." %}</p>
+  <p>{% trans "Linked project spaces allow you to:" %}</p>
+  <ol>
+    <li>{% trans "Push content to a downstream project space" %}</li>
+    <li>{% trans "Pull content from an upstream project space" %}</li>
+  </ol>
   <p>
     {% blocktrans %}
-      Get started by creating a link between two projects, allowing you to push and pull content.<br/>
-      If you want to pull content from another project into this project, make it a downstream project.<br/>
-      <a href="https://confluence.dimagi.com/display/commcarepublic/Linked+Project+Spaces" target="_blank">Learn more</a> about linked projects.
+      <a href="https://confluence.dimagi.com/display/commcarepublic/Linked+Project+Spaces" target="_blank">Learn more</a> about Linked Project Spaces.
```

Comment:
@biyeun I know you've said the use `blocktrans` for instances like these in the past, but you've also said to avoid using `html` elements within a `blocktrans` block. Is this the best way to go about doing this? Mainly changed to handle indentation.

Reply:
Offlined with Biyeun and she is good with this style. This avoids the risk of translators removing html by accident.